### ISSUE TITLE
Error when initializing `TheGreatBrainRobbery`

### ISSUE THREAD

**Issue by Felanbird** [Zoinkwiz/quest-helper#1035]
Noticed while in `debug` while messing with another plugin, happens when loading plugins before logging in
```
2023-01-14 14:59:09 [Client] ERROR n.r.client.callback.ClientThread - Exception in invoke
java.lang.NullPointerException: Cannot read the array length because "client.jp" is null
        at client.getVarbitValue(client.java:9184)
        at client.getVarbitValue(client.java:58466)
        at com.questhelper.quests.thegreatbrainrobbery.TheGreatBrainRobbery.setupRequirements(TheGreatBrainRobbery.java:272)
        at com.questhelper.QuestHelperPlugin.lambda$startUp$0(QuestHelperPlugin.java:318)
        at java.base/java.util.HashMap.forEach(HashMap.java:1421)
        at com.questhelper.QuestHelperPlugin.lambda$startUp$1(QuestHelperPlugin.java:316)
        at net.runelite.client.callback.ClientThread.lambda$invokeLater$1(ClientThread.java:80)
        at net.runelite.client.callback.ClientThread.invokeList(ClientThread.java:119)
        at net.runelite.client.callback.ClientThread.invoke(ClientThread.java:101)
        at net.runelite.client.callback.Hooks.tick(Hooks.java:213)
        at client.ac(client.java:31956)
        at client.an(client.java)
        at an.ao(an.java:390)
        at an.run(an.java:369)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by iProdigy** [comment]
Would be fixed by https://github.com/Zoinkwiz/quest-helper/pull/1034

**Comment by Felanbird** [comment]
Fixed in #1034
